t/lib/t/MusicBrainz/Server/Controller/WS/2/BrowseRelease.pm

```diff
@@ -382,6 +382,79 @@ ws_test 'browse releases via recording',
     </release-list>
 </metadata>';
 
+ws_test 'browse releases via recording, with recording rels',
+    '/release?inc=recordings+artist-rels+recording-level-rels&status=official&recording=c43ee188-0049-4eec-ba2e-0385c5edd2db' =>
+    '<?xml version="1.0" ?>
+<metadata xmlns="http://musicbrainz.org/ns/mmd-2.0#">
+    <release-list count="1">
+        <release id="ec0d0122-b559-4aa1-a017-7068814aae57">
+            <title>Soup</title>
+            <status id="4e304316-386d-3409-af2e-78857eec5cfe">Official</status>
+            <quality>normal</quality>
+            <text-representation>
+            <language>eng</language>
+            <script>Latn</script>
+            </text-representation>
+            <barcode>0208311348266</barcode>
+            <cover-art-archive>
+            <artwork>false</artwork>
+            <count>0</count>
+            <front>false</front>
+            <back>false</back>
+            </cover-art-archive>
+            <medium-list count="1">
+            <medium>
```

Comment:
nitpick, but some of the indentation here is off (text-representation, cover-art-archive, medium-list, pregap, etc.)